tests/bc_2_6_0/src/test/java/org/apache/pulsar/tests/integration/PulsarContainer.java

```diff
@@ -29,7 +29,14 @@
 
     public static final int PULSAR_PORT = 6650;
     public static final int BROKER_HTTP_PORT = 8080;
-    public static final String DEFAULT_IMAGE_NAME = "apachepulsar/pulsar-test-latest-version:latest";
+    public static final String DEFAULT_IMAGE_NAME;
+    static {
+        String containerName = System.getenv("PULSAR_CONTAINER_IMAGE_NAME");
```

Comment:
can we use System.getenv(string, defaultValue) ?

Reply:
there isn't such a method. However it seems that `System.getenv().getOrDefault(string, defaultValue)` is what can be done.

Reply:
Fixed